cmd/minishift/cmd/addon/addon_apply.go

```diff
@@ -75,15 +77,22 @@ func runApplyAddon(cmd *cobra.Command, args []string) {
 		atexit.ExitWithMessage(1, fmt.Sprintf("Error getting IP: %s", err.Error()))
 	}
 
-	ocPath := minishiftConfig.InstanceConfig.OcPath
 	routingSuffix := viper.GetString(routingSuffix)
-	kubeConfigPath := constants.KubeConfigPath
 	sshCommander := provision.GenericSSHCommander{Driver: host.Driver}
+	ocPath := minishiftConfig.InstanceConfig.OcPath
+	kubeConfigPath := constants.KubeConfigPath
```

Comment:
Unless, `ocPath` or `kubeConfigPath` is used multiple times in later part of this file, we can directly pass these values.